### PR TITLE
Update repository with new add-on version

### LINKED ISSUE #3: Rename the addon to "Just Delete Me".
T/O

### LINKED ISSUE #4: Firefox Addon Resurrection
To my knowledge, per jdm-contrib/jdm#181, this addon does not work currently, and will not work due to the way Firefox has changed their handling of extensions. With that said, what we have present in the Chrome extension repo may actually work with little-to-no changes for Firefox. (research needed)

### LINKED ISSUE #6: mdn dead
would you just bring it back instead of giving 404? pls...

### LINKED ISSUE #7: I made a new extension
Since this repo seems inactive and the codebase isn't super clean, I made an extension that uses JustDeleteMe.

https://github.com/alexanderepolite/JustDeleteMe-extension

Let me know what you think!

### PR DESCRIPTION
**Pull request by AlexanderEpolite**
This is the big PR that was delayed for several months.

Closes #7 (for obvious reasons).
Closes #6 as the link is no longer in the README.
Closes #4
Closes #3

Not sure about issue number 2, let me know about that one.  The README might need some updating, so feel free to suggest changes.

### CODE REVIEW

**Comment by AlexanderEpolite** [comment]
Additionally, the URL in the repository will need to be changed: https://addons.mozilla.org/en-US/firefox/addon/justdeleteme/

File: README.md [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
 
-Provides link on a site to where to delete account and informs you how hard it is to delete your account.
+This extension provides details on how to delete your accounts for websites, along with how difficult it is.
 
-Information about how to delete an account is pulled daily from JustDelete.me ( http://justdelete.me/ ). 
+The extension icon will change depending on the difficulty, ranging from easy to impossible.
 
-This extension will add a traffic light icon to your urlbar indicating the difficulty in removing an account on the website you're visiting is.
+## Download
+You can download the extension [![at this link](https://ffp4g1ylyit3jdyti1hqcvtb-wpengine.netdna-ssl.com/addons/files/2015/11/get-the-addon.png)](https://addons.mozilla.org/en-US/firefox/addon/justdeleteme/).
```
Comment (tupaschoal) [inline README.md]: I tried viewing the readme as `md` and seems like the image is broken.

File: js/background.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+
+const data_url = "https://raw.githubusercontent.com/jdm-contrib/jdm/master/_data/sites.json";
+
+//taken from https://github.com/jdm-contrib/justdelete.me-firefox-addon/blob/master/lib/main.js
```
Comment (tupaschoal) [inline js/background.js]: I think we can get away with this comment, since we intend to merge :P

File: js/background.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+(async () => {
+    console.log(`Started background server.`);
+    
+    //fetch the data
+    const json = await (await fetch(data_url)).json();
+    
+    /**
+     * Set the icon.
+     *
+     * @param icon {"easy" | "medium" | "hard" | "impossible" | "unknown"}
```
Comment (tupaschoal) [inline js/background.js]:
```suggestion
     * @param icon {"easy" | "medium" | "hard" | "impossible" | "limited" | "unknown"}
```

File: js/background.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                128: `res/icons/${icon}.png`,
+            },
+        });
+    }
+    
+    /**
+     * Get the hostname from a URL.
+     * 
+     * Parodied from https://github.com/jdm-contrib/justdelete.me-firefox-addon/blob/master/lib/main.js
+     *
```
Comment (tupaschoal) [inline js/background.js]:
```suggestion
```

outdated comment

File: js/background.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        }
+        
+        return new URL(url).hostname;
+    }
+    
+    /**
+     * Search for the site and return the difficulty
+     * 
+     * @param domain {string}
+     * @returns {"easy" | "medium" | "hard" | "impossible" | undefined} the difficulty, or undefined if there is none.
```
Comment (tupaschoal) [inline js/background.js]:
```suggestion
     * @returns {"easy" | "medium" | "hard" | "impossible" | "limited" | undefined} the difficulty, or undefined if there is none.
```

File: pub/index.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        }
+    </style>
+</head>
+<body>
+<div class="title">
+    <h1>JustDeleteMe</h1>
+</div>
+<div class="unofficial_warning">
+    <h6>Unofficial JustDeleteMe.xyz browser extension.</h6>
+</div>
```
Comment (tupaschoal) [inline pub/index.html]:
```suggestion
```

We won't be needing that :)

**Review by tupaschoal — COMMENTED** [review]
Thanks for coming back! I have just a few simple notes.

**Comment by AlexanderEpolite** [comment]
@tupaschoal The commits are unsigned because I am on my laptop right now and do not have access to my private key.  I have resolved the notes you made, let me know if there is anything else that needs to be changed before the merge.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Everything here looks fine @alexanderepolite , thanks!

I've invited you to the organization to have access to the extension repos. I think you might need to do some updating on the add-on webpage itself to point to its new home.

**Comment by AlexanderEpolite** [comment]
Everything looks good!  Can you change the repository URL to https://addons.mozilla.org/en-US/firefox/addon/justdeleteme/ @tupaschoal ?  I do not have permission to change this.

**Comment by tupaschoal** [comment]
Done!

**Pull request merged**